pkg/dependency/parser/python/pip/parse.go

```diff
@@ -25,15 +25,30 @@ const (
 )
 
 type Parser struct {
-	logger *log.Logger
+	logger        *log.Logger
+	useMinVersion bool
 }
 
-func NewParser() *Parser {
+func NewParser(useMinVersion bool) *Parser {
 	return &Parser{
-		logger: log.WithPrefix("pip"),
+		logger:        log.WithPrefix("pip"),
+		useMinVersion: useMinVersion,
 	}
 }
 
+func splitLine(line string) []string {
```

Comment:
I thought about `*`.
```
== 3.1.*: any version that starts with 3.1. Equivalent to the ~=3.1.0 compatible release clause.
```

We show minimal version, so it looks like we can replace `*` to `0`.
wdyt?

Reply:
Agree!

Reply:
replacing a trailing `*` is added.